src/sentry/migrations/0344_create_organizationmapping.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.28 on 2022-11-08 05:02
+
+import django.utils.timezone
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0341_reconstrain_savedsearch_pinning_fields"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="OrganizationMapping",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                (
+                    "organization_id",
+                    sentry.db.models.fields.bounded.BoundedBigIntegerField(db_index=True),
+                ),
+                ("slug", models.SlugField(unique=True)),
+                ("created", models.DateTimeField(default=django.utils.timezone.now)),
+                ("stripe_id", models.CharField(max_length=255, db_index=True)),
```

Comment:
So glad this is coming!  Actually, this table can power some other organization service logic.  `get_organizations` as written cannot work without this table.

Reply:
To that point, can we also have the `name` attribute in this table?  The reason is that we actually end up needing to get all organization's slugs and names associated with auth identities, having it in the mapping makes this possible without "spanning out": https://github.com/getsentry/sentry/pull/41592/files#diff-31c94c3edd54a4fd9293854c54ffae294bd0436926e455c1c1fc1b577fd65d7eR222

Reply:
We can certainly denormalize that too, but it expands the scope a bit. I'll add it to the model and maybe we can tackle the actual updates to it later

Reply:
Sure!